pkgs/tools/misc/xjobs/default.nix

```
@@ -0,0 +1,65 @@
+{ lib, stdenv, fetchurl,
+flex, ncurses, which
+}:
+
+stdenv.mkDerivation rec {
+  version = "20200726";
+  pname = "xjobs";
```

```suggestion
  pname = "xjobs";
  version = "20200726";
```
This is normally the default order, not a big deal but whatever